lib/gringotts/gateways/sagepay.ex

```diff
@@ -34,68 +23,71 @@ defmodule Gringotts.Gateways.Sagepay do
 
   --------------------------------------------------------------------------------
 
-  > List features that have been implemented, and what "actions" they map to as
-  > per the sagepay gateway docs.
-  > A table suits really well for this.
-
-  ## Optional or extra parameters
 
   Most `Gringotts` API calls accept an optional `Keyword` list `opts` to supply
   optional arguments for transactions with the gateway.
 
-  > List all available (ie, those that will be supported by this module) keys, a
-  > description of their function/role and whether they have been implemented
-  > and tested.
-  > A table suits really well for this.
+  The following features of SagePay are implemented:
 
-  ## Registering your sagepay account at `Gringotts`
+  | Action                       | Method        | 
+  | ------                       | ------        | 
+  | Authorize                    | `authorize/3` | 
+  | Release                      | `capture/3`   | 
+  | Refund                       | `refund/3`    | 
+  | Reversal                     | `void/2`      | 
+  | Purchase                     | `purchase/3`  | 
 
-  Explain how to make an account with the gateway and show how to put the
-  `required_keys` (like authentication info) to the configuration.
+  [home]: sagepay.co.uk
+  [docs]: integrations.sagepay.co.uk
 
-  > Your Application config would look
-  > something like this:
-  > 
-  >     config :gringotts, Gringotts.Gateways.Sagepay,
 
+  ## Registering your SagePay account at `Gringotts`
+
+  After [making an account on SagePay][dashboard], head to the dashboard and find
+  your account "secrets" in the `Sub-Accounts > Overview` section.
+
+  Here's how the secrets map to the required configuration parameters for SagePay:
+
+  | Config parameter        | SagePay secret         |
+  | ------------------------| -----------------------|
+  | `:auth_id`              | **Authorization Id**   |
+  | `:vendor_name`          | **Name of a merchant** |
+
+  Your Application config **must include the `:Authorization Id`, `:Merchant name`
+  fields** and would look something like this:
+
+      config :gringotts, Gringotts.Gateways.SagePay,
+        auth_id: "your_secret_user_id",
+        vendor_name: "name_of_vendor"
+        
 
   ## Scope of this module
 
-  > It's unlikely that your first iteration will support all features of the
-  > gateway, so list down those items that are missing.
+  * SagePay does not process money in cents, and the `amount` is converted to integer.
```

Comment:
What is "rendered" in the authorize request if I wish to make a payment for $3.14, `"314"` or `"3.14"`?